deployment/params.py

```diff
@@ -308,21 +300,52 @@ def resolve(self, contract_name: str) -> OrderedDict:
         return result
 
 
-class Deployer:
+class Transactor:
```

Comment:
Consider repurposing this class to wrap the ape project instead, so you can maintain the same ape syntax for transactions (plus any additional processing you want):

```py
from nucypher.ape import project
...
project.Contract[-1].method(param1, param2, sender=alice, interactive=True)
```

and in `nucypher.ape`:
```py
from ape import project

project = Transactor(project)
```
In any case, not a blocker for this PR.